README.md

```diff
@@ -283,3 +283,12 @@ Object returned for getMeta and list methods, object has the following propertie
 	'modified': Wed Jan 01 2014 15:11:07 GMT+0100 (W. Europe Standard Time)
 }
 ```
+
+
+# License
+MIT
+
+# Contributing
```

Comment:
Seems there's a mixed standard, some repos have it as the second to last (maybe move licensing below) and some have it at the top. Most have a `CONTRIBUTING.md`-file in the root of the project.